plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -108,6 +108,83 @@ func addClusterRoleBindingLabel(rolebindings []rbacv1.ClusterRoleBinding) {
 	return
 }
 
+func viewRules() []rbacv1.PolicyRule {
```

Comment:
the code movement makes it really hard to see what this PR is changing

can you split the move into a standalone commit, so the delta is in its own commit and is easy to see?

Reply:
move existing rules to a new function - https://github.com/kubernetes/kubernetes/pull/130738/commits/00101a5725991231e1f846151f7f3a2880aa475d
add DRA to the edit role - https://github.com/kubernetes/kubernetes/pull/130738/commits/1c86dfa1a87d57975eaab27d18915755670f96ec